Library/Homebrew/livecheck/strategy/sparkle.rb

```diff
@@ -86,6 +86,16 @@ def self.item_from_content(content)
 
             next if (os = enclosure&.attr("os")) && os != "osx"
 
+            if OS.mac? && (minimum_system_version = (item > "minimumSystemVersion").first&.text&.strip)
+              macos_minimum_system_version = begin
+                MacOS::Version.new(minimum_system_version).strip_patch
+              rescue MacOSVersionError
+                nil
+              end
+
+              next if MacOS.version < macos_minimum_system_version
```

Comment:
```suggestion
              next if macos_minimum_system_version&.prerelease?
```

#11581 is now merged, so we can simply use `OS::Mac::Version#prerelease?` to determine if `macos_minimum_system_version` is for an unstable version. This simply depends on the information in `OS::Mac::Version` being current and won't require us to update `Sparkle` when a new macOS version appears.